jamaljsr/polar
# Feat/persistent-docker-network

-NewNetwork now has an input to create/attach to an external docker network
-NetworkActions now has a modal for the user to
create/attach/clear to an external docker network.

Closes #(issue number goes here)
525
https://github.com/jamaljsr/polar/issues/525

### Description

Allows for polar to attach or create an external docker network.

### Steps to Test

1. Create a new network.
2. Fill out the Docker External Network field
3. Check the docker-compose file or `docker network ls`

1. In a current network, go to docker options in the hamburger menu
of NetworkActions.
2. In the modal, name your docker network. Hit okay
3. Check the docker-compose file or `docker network ls`.

A status icon is also visible.
